manifests/init.pp

```diff
@@ -50,9 +50,12 @@
   $system_auth_ac_password_lines       = undef,
   $system_auth_ac_session_lines        = undef,
   $vas_major_version                   = '4',
+  $manage_nsswitch                     = true,
 ) {
 
```

Comment:
needs validation. Puppet's eco system also has a history of turning things into strings, so if `'true'` or `'false'` get passed, we need to convert to booleans.

Check out this example

https://github.com/ghoneycutt/puppet-module-pam/blob/master/manifests/init.pp#L1097-1102